src/components/SecondaryPanes/Breakpoints/Breakpoints.css

```diff
@@ -34,57 +37,75 @@
 
 .breakpoints-list .breakpoint-heading,
 .breakpoints-list .breakpoint {
-  font-size: 12px;
   color: var(--theme-content-color1);
   position: relative;
-  transition: all 0.25s ease;
   cursor: pointer;
 }
 
 .breakpoints-list .breakpoint-heading,
 .breakpoints-list .breakpoint,
 .breakpoints-exceptions,
 .breakpoints-exceptions-caught {
-  padding: 0.25em 1em;
+  padding-inline-start: 16px;
+  padding-inline-end: 12px;
 }
 
 .breakpoints-exceptions {
-  padding-bottom: 0.5em;
-  padding-top: 0.5em;
+  padding-bottom: 3px;
+  padding-top: 3px;
   user-select: none;
 }
 
 .breakpoints-list .breakpoint {
-  min-height: var(--breakpoint-expression-height);
   overflow: hidden;
+  padding-top: 2px;
+  padding-bottom: 2px;
+}
+
+.breakpoints-list .breakpoint:last-child {
+  margin-bottom: 4px;
 }
 
 .breakpoints-exceptions-caught {
-  padding: 0 1em 0.5em 3em;
-  margin-top: -0.25em;
+  padding-bottom: 3px;
+  padding-top: 3px;
+  padding-inline-start: 36px;
 }
 
 html[dir="rtl"] .breakpoints-exceptions-caught {
   padding: 0 3em 0.5em 1em;
```

Comment:
We can probably remove this rtl style, since we use `padding-inline-start` above.

Reply:
Removed.